src/window/window_wrapper.rs

```diff
@@ -411,6 +411,17 @@ impl WinitWindowWrapper {
         skia_renderer.swap_buffers();
         if self.ui_state == UIState::FirstFrame {
             skia_renderer.window().set_visible(true);
+            let is_maximized = matches!(self.initial_window_size, WindowSize::Maximized);
+            let is_x11 = matches!(
+                skia_renderer.window().window_handle().unwrap().as_raw(),
```

Comment:
Please let me know if unwrapping the window handle is ok at this point, I guess it must be?